packages/renderer/src/lib/kube/KubernetesIcon.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+import type { Component } from 'svelte';
+
+import ConfigMapSecretIcon from '../images/ConfigMapSecretIcon.svelte';
+import CronJobIcon from '../images/CronJobIcon.svelte';
+import DeploymentIcon from '../images/DeploymentIcon.svelte';
+import IngressRouteIcon from '../images/IngressRouteIcon.svelte';
+import JobIcon from '../images/JobIcon.svelte';
+import KubeIcon from '../images/KubeIcon.svelte';
+import NodeIcon from '../images/NodeIcon.svelte';
+import PodIcon from '../images/PodIcon.svelte';
+import PvcIcon from '../images/PVCIcon.svelte';
+import ServiceIcon from '../images/ServiceIcon.svelte';
```

Comment:
nit: as other PR, prefer the `/@...` import rather than ../ (in case of moving this file to another folder, imports will remain the same)

Reply:
Yup, I was just doing copy/paste without cleanup. Fixed.